src/Adapter/GeneralConfiguration.php

```diff
@@ -44,24 +45,38 @@ class GeneralConfiguration implements DataConfigurationInterface
      */
     private $cookie;
 
-    public function __construct(Configuration $configuration, Cookie $cookie)
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    public function __construct(Configuration $configuration, Cookie $cookie, bool $isDebug)
     {
         $this->configuration = $configuration;
         $this->cookie = $cookie;
+        $this->isDebug = $isDebug;
     }
 
     /**
      * {@inheritdoc}
      */
     public function getConfiguration()
     {
-        return [
+        $configuration = [
             'check_modules_update' => $this->configuration->getBoolean('PRESTASTORE_LIVE'),
             'check_ip_address' => $this->configuration->getBoolean('PS_COOKIE_CHECKIP'),
             'front_cookie_lifetime' => $this->configuration->get('PS_COOKIE_LIFETIME_FO'),
             'back_cookie_lifetime' => $this->configuration->get('PS_COOKIE_LIFETIME_BO'),
             'cookie_samesite' => $this->configuration->get('PS_COOKIE_SAMESITE'),
         ];
+        if ($this->isDebug) {
```

Comment:
Is there a reason to do it only in debug mode? :thinking:

Reply:
It's just useless option for regular merchants, as @Progi1984 said `channel` is used only for in-house PrestaShop modules at this moment, I was the one suggesting that it should be available only in dev mode 😝

Reply:
Many regular merchants are using the BO in dev mode without understand why the debug mode is loaded :sweat_smile: 

If it's not a development feature, the isDebug is not required.

Reply:
yeah but is it safe to allow people to change anything to "alpha" or "beta" on production site? :) this is tricky scenario because this option is only useful for modules like `ps_checkout` and few others, this is why I suggested to put in in dev mode, it's... for dev purposes for `ps_checkout` for example :)

Reply:
I think it comes from those specs: https://github.com/PrestaShop/prestashop-specs/pull/169/files

Reply:
Correct @atomiix